mandatory/2-function-creation.js

```diff
@@ -15,26 +23,58 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice>200){
+  return totalPrice *0.9;
+}else if (totalPrice<200){
+  return totalPrice * 0.95;
+ }
+}
```

Comment:
Nice one! ☝️ 

This is exactly how the `else if` statement should be used, because we have a third condition where the `totalPrice` could be exactly equal to 200, but we haven't been requested to make a discount when the totalPrice is 200!

We should also `return totalPrice` just to ensure we have all cases covered (even though i'm aware there are no tests for this 😅  feel free to add one and try it out ^^)